tests/unit/testplan/runnable/interactive/test_reloader.py

```diff
@@ -269,13 +272,8 @@ def test_test_refresh(mock_reload_env):
     test.cfg.suites[0].__module__ = "mod_a"
     mock_stat.modified_files = {MOCK_MODULES["mod_a"]}
 
-    set_testsuite_testcases = (
-        "testplan.testing.multitest.suite.set_testsuite_testcases"
-    )
-    with mock.patch(set_testsuite_testcases) as mock_set_testcases:
-        reload_obj.reload(tests=[test])
-        mock_reload.assert_called_once_with(MOCK_SYSMODULES["mod_a"])
-        mock_set_testcases.assert_called_once_with(test.cfg.suites[0])
```

Comment:
It seems we are losing test coverage? Below is no longer checked.

mock_set_testcases.assert_called_once_with(test.cfg.suites[0])